test_util/azure_test_driver.py

```diff
@@ -58,7 +61,16 @@ def get_value(template_parameter):
     return(get_env_params()[template_parameter]['value'])
 
 
-def run():
+def get_test_config():
+    add_env = {}
```

Comment:
Its now passed via TEST_ADD_ENV_DCOS_AUTH_ENABLED but we can specifically handle this case. Every option felt awkward so its all the same to me